rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java

```diff
@@ -572,6 +572,11 @@ public void onFailure(Throwable caught) {
                 } else {
                     error("Failed to fetch RM State: " + JSONUtils.getJsonErrorMessage(caught));
                 }
+                if (caught.getMessage().contains("HTTP 401 Unauthorized")) {
+                    rmPage.showErrorLabel("You have been disconnected from the server. Reload the page and log into the portal.");
```

Comment:
You have been disconnected from the server. Reload the page in order to log into the portal again.